v0/destinations/posthog/data/PHPropertiesConfig.json

```diff
@@ -140,5 +128,37 @@
     "destKey": "$app_build",
     "sourceKeys": "context.app.build",
     "isMobile": true
+  },
+  {
+    "destKey": "$viewport_height",
+    "sourceKeys": ["traits.viewport.height", "context.traits.viewport.height"]
+  },
+  {
+    "destKey": "$viewport_width",
+    "sourceKeys": ["traits.viewport.width", "context.traits.viewport.width"]
```

Comment:
```suggestion
    "sourceKeys": ["context.screen.width"]
```

Reply:
@utsabc we already have these sourceKeys for $screen_width and $screen_height properties